src/lib.rs

```diff
@@ -1123,6 +1123,19 @@ pub trait Itertools : Iterator {
     ///     vec![2, 3, 4],
     ///     ]);
     /// ```
+    ///
+    /// Note: Combinations does not take into accout the equality of the inner values.
```

Comment:
Spelling nit: `accout` -> `account`

Wording nit: `the inner values` -> `the iterated values`